.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Extractors/ChineseMergedExtractorConfiguration.cs

```diff
@@ -219,5 +233,35 @@ private void AddTo(List<ExtractResult> dst, List<ExtractResult> src)
                 }
             }
         }
+
+        // Avoid adding mod for ambiguity cases, such as "从" in "从 ... 到 ..." should not add mod
+        // TODO: Revise PotentialAmbiguousRangeRegex to support cases like "从2015年起，哪所大学需要的分数在80到90之间"
```

Comment:
How big do you estimate is the effort to cover this case?

Reply:
Currently, in En "**from** 2015, I have wanted **to** buy a new car" it will only get "2015", not from 2015.
I'm not sure how to fix this bug.